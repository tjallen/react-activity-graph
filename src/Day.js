/* eslint-disable no-console */
import React, { Component } from 'react';
// import PropTypes from 'prop-types';

export default class Day extends Component {
  // static propTypes = {
  // }
  // static defaultProps = {
  // }
  constructor(props) {
    super(props);
    this.state = {
    };
  }
  render() {
    const styles = {
      fontSize: '8px',
      border: '1px solid white',
      height: '10px',
      width: '10px',
    };
    return (
      <div style={styles}>
        {this.props.children}
      </div>
    );
  }
}
