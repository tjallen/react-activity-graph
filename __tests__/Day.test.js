import React from 'react';
import Day from '../src/Day';
import renderer from 'react-test-renderer';

const expected = {
  key: 1,
  value: 99,
  date: '1901-01-01',
  formattedDate: '1st Jan, 1901',
  maxValue: 100,
  color: '#000000',
};

const component = renderer.create(
  <Day
    key={expected.key}
    value={expected.value}
    date={expected.date}
    formattedDate={expected.formattedDate}
    maxValue={expected.maxValue}
    color={expected.color}
  />
);

test('Day snapshot test', () => {
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

