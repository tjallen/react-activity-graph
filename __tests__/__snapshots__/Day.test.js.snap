// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Day snapshot test 1`] = `
<div
  className="sc-htpNat lcFUQH"
>
  <div
    className="sc-bwzfXH cZKcsL"
  >
    <div
      className="sc-bdVaJa dOxDA"
    >
      99
       on 
      1st Jan, 1901
    </div>
  </div>
</div>
`;
